components/sup/src/manager/service/mod.rs

```diff
@@ -85,6 +85,7 @@ pub struct Service {
     config_renderer: CfgRenderer,
     health_check: HealthCheck,
     initialized: bool,
+    post_run: bool,
```

Comment:
This indicates whether the last run of the `post_run` hook was successful, correct?

If so, I'd recommend having the name of this field reflect that.

Reply:
I ended up on post_run_exit_code - happy to change as this seems a bit weak.